<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckList</class>
 <widget class="QWidget" name="CheckList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameTristate">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Selection means</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cBNone">
        <property name="toolTip">
         <string>Do not export this items to user defined contact list</string>
        </property>
        <property name="text">
         <string> -&gt; Unselected items</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cBSelect">
        <property name="toolTip">
         <string>Marked items will be saved in users contact list</string>
        </property>
        <property name="text">
         <string> -&gt; Selected items</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cBRequired">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Contacts that includes this item will be saved in users contact list</string>
        </property>
        <property name="text">
         <string> -&gt; Required items</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select items  </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tBCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/img/close.png</normaloff>:/img/img/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>33</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tBSave">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/img/dialog-ok-apply.png</normaloff>:/img/img/dialog-ok-apply.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>33</width>
         <height>33</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
